gravitee-apim-console-webui/src/management/api/portal/details/api-portal-details-danger-zone/api-portal-details-danger-zone.component.ts

```diff
@@ -65,6 +68,21 @@ export class ApiPortalDetailsDangerZoneComponent implements OnChanges, OnDestroy
   };
   public isReadOnly = false;
 
+  public get shouldUpgrade$() {
+    const api = this.api as ApiV4;
+    if (api.type === 'PROXY') {
+      return false;
+    }
+    return this.licenseService?.isMissingPack$(Pack.EVENT_NATIVE);
```

Comment:
What about the case where I do not have the pack but just the feature(s)? Is that even possible?

Reply:
@aelamrani just discussed with the en team. It happens that enabling any en feature implies having en-reactor-message in the license. fix is on the way.